src/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeSystem/Handles.TypeSystem.cs

```diff
@@ -2341,9 +2341,9 @@ internal StringHandle SuffixRaw(int prefixByteLength)
 
         public static implicit operator Handle(StringHandle handle)
         {
-            // VTT... -> V111 10TT
+            // VTTx xxxx xxxx xxxx  xxxx xxxx xxxx xxxx -> V111 10TT
             return new Handle(
-                (byte)((handle._value & HeapHandleType.VirtualBit) >> 24 | HandleType.String | (handle._value & StringHandleType.NonVirtualTypeMask) >> 26),
+                (byte)((handle._value & HeapHandleType.VirtualBit) >> 24 | HandleType.String | (handle._value & StringHandleType.NonVirtualTypeMask) >> HeapHandleType.OffsetBitCount),
```

Comment:
Yeah. I noticed since I use casting heavily in MDV and it was showing garbage...